client/src/app/pages/applications/components/bulk-copy-assessment-review-form/bulk-copy-assessment-review-form.tsx

```diff
@@ -81,7 +81,11 @@ export const BulkCopyAssessmentReviewForm: React.FC<
 
   const [confirmationAccepted, setConfirmationAccepted] = useState(false);
 
-  const { applications, isFetching, fetchError } = useFetchApplications();
+  const {
+    data: applications,
```

Comment:
Any reason to create these aliases rather than export the explicit name of the entity we are dealing with?

Reply:
Here we're piggy backing on React Query default returns. We can destructure the returned object where it's consumed. Later on it should help being more generic with queries.